### PR TITLE
Tao 1726 localised parse numbers

### PR DESCRIPTION
**Pull request by Alroniks**
Related to https://github.com/oat-sa/extension-tao-itemqti/pull/500
Related to https://oat-sa.atlassian.net/browse/TAO-1726
Comments by task here: https://oat-sa.atlassian.net/browse/TAO-1723

### CODE REVIEW

File: views/js/util/locale.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+define(['module'], function (module) {
+    'use strict';
+
+    /**
+     * Util object for manipulate locale dependent data
+     * @exports util/locale
+     */
+    return {
+
+        config: module.config(),
```
Comment (ssipasseuth) [inline views/js/util/locale.js]: I would suggest to set the variable config as private and add a setter and getter instead. This way we can better ace and trcontrol the modification on the config.

Comment (Alroniks) [inline views/js/util/locale.js]: I agree. @ssipasseuth Can you share examples or any article about how correct create setter and getter on javascript?

Comment (ssipasseuth) [inline views/js/util/locale.js]:
```javascript
define(function(){
       //private var, accessible only in the scope
       var privateVar = null;
       
       //return object with public methods
       return {
            getPrivateVar : function(){
                return privateVar;
            },
            setPrivateVar : function(newValue){
                privateVar = newValue
            }
       };
    });

**Comment by ssipasseuth** [comment]
This is a bug fix for project PARCC. And I just checked with @jbout, the fix needs to be done to the master branch for extensions that are not parcc related. So could you please create a branch from master, cherry pick your changes and create a PR to master.

**Comment by Alroniks** [comment]
@ssipasseuth Fixed your notises and created PR for master branch https://github.com/oat-sa/tao-core/pull/711

**Comment by ssipasseuth** [comment]
Replaced with https://github.com/oat-sa/tao-core/pull/714

**Pull request closed without merging**